src/command_modules/azure-cli-vm/HISTORY.rst

```diff
@@ -2,6 +2,8 @@
 
 Release History
 ===============
+* vm list-skus: --all should be a flag instead of an argument requiring values
```

Comment:
I would reword this. 

`* vm list-skus: Allow use of `--all` in place of `--all true`